pkg/clean/clean.go

```diff
@@ -25,19 +27,35 @@ var log = logger.Log
 // including security policy, static route, subnet, subnet port, subnet set, vpc, ip pool, nsx service account
 // it is usually used when nsx-operator is uninstalled and remove all the resources created by nsx-operator
 // return error if any, return nil if no error
+// the error roughly are:
+// 1. failed to validate config
+// 2. failed to get nsx client
+// 3. failed to initialize cleanup service
+// 4. failed to clean up specific resource
 func Clean(cf *config.NSXOperatorConfig) error {
 	log.Info("starting NSX cleanup")
 	if err := cf.ValidateConfigFromCmd(); err != nil {
 		return fmt.Errorf("failed to validate config: %w", err)
 	}
 	if cleanupService, err := InitializeCleanupService(cf); err != nil {
-		return fmt.Errorf("failed to initialize cleanup service: %w", err)
+		return err // failed to get nsx client
```

Comment:
Why remove the context of the previous note?

Reply:
I think it is duplicated with line 43, so remove it.